### ISSUE TITLE
Gradle exception: Failed to parse React Native CLI configuration

### ISSUE THREAD

**Issue by fabiocosta88** [react-native-community/cli#863]
## Environment

System:
    OS: Windows 10
    CPU: (4) x64 Intel(R) Core(TM) i5-3570 CPU @ 3.40GHz
    Memory: 1.47 GB / 7.96 GB
  Binaries:
    Node: 12.8.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.19.1 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.10.2 - C:\Program Files\nodejs\npm.CMD
  SDKs:
    Android SDK:
      API Levels: 25, 27, 28, 29
      Build Tools: 25.0.3, 28.0.3, 29.0.2
      System Images: android-28 | Google Play Intel x86 Atom, android-29 | Google APIs Intel x86 Atom, android-29 | Google Play Intel x86 Atom
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native: 0.61.4 => 0.61.4

## Description

  Simply trying to run the react-native run-android command returns me this React Native CLI-related error.

`
info Running jetifier to migrate libraries to AndroidX. You can disable it using "--no-jetifier" flag.
Jetifier found 1032 file(s) to forward-jetify. Using 4 workers...
info Starting JS server...
info Installing the app...
:ReactNative:Failed to parse React Native CLI configuration: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object

The current character read is 'E' with an int value of 69
Unable to determine the current character, it is not a string, number, array, or object
line number 1
index number 0
Error: EPERM: operation not permitted, scandir 'C:/Users/Master/Desktop/proj/UltimaTentativa/android/app/build/intermediates/signing_config/debug/out/signing-config.json'    at Object.readdirSync (fs.js:795:3)    at GlobSync._readdir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:288:41)    at GlobSync._readdirInGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:267:20)    at GlobSync._readdir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:276:17)    at GlobSync._processReaddir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:137:22)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:132:10)    at GlobSync._processGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:380:10)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:130:10)    at GlobSync._processGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:383:10)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:130:10)
^

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\@react-native-community\cli-platform-android\native_modules.gradle' line: 200

* What went wrong:
A problem occurred evaluating script.
> Failed to parse React Native CLI configuration. Expected running 'npx.cmd --quiet --no-install react-native config' command from 'C:\Users\Master\Desktop\proj\UltimaTentativa' directory to output valid JSON, but it didn't. This may be caused by npx resolving to a legacy global react-native binary. Please make sure to uninstall any global 'react-native' binaries: 'npm uninstall -g react-native react-native-cli' and try again

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

error Failed to install the app. Make sure you have the Android development environment set up: https://facebook.github.io/react-native/docs/getting-started.html#android-development-environment. Run CLI with --verbose flag for more details.
Error: Command failed: gradlew.bat app:installDebug -PreactNativeDevServerPort=8081
:ReactNative:Failed to parse React Native CLI configuration: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object

The current character read is 'E' with an int value of 69
Unable to determine the current character, it is not a string, number, array, or object
line number 1
index number 0
Error: EPERM: operation not permitted, scandir 'C:/Users/Master/Desktop/proj/UltimaTentativa/android/app/build/intermediates/signing_config/debug/out/signing-config.json'    at Object.readdirSync (fs.js:795:3)    at GlobSync._readdir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:288:41)    at GlobSync._readdirInGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:267:20)    at GlobSync._readdir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:276:17)    at GlobSync._processReaddir (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:137:22)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:132:10)    at GlobSync._processGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:380:10)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:130:10)    at GlobSync._processGlobStar (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:383:10)    at GlobSync._process (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\glob\sync.js:130:10)
^

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\@react-native-community\cli-platform-android\native_modules.gradle' line: 200

* What went wrong:
A problem occurred evaluating script.
> Failed to parse React Native CLI configuration. Expected running 'npx.cmd --quiet --no-install react-native config' command from 'C:\Users\Master\Desktop\proj\UltimaTentativa' directory to output valid JSON, but it didn't. This may be caused by npx resolving to a legacy global react-native binary. Please make sure to uninstall any global 'react-native' binaries: 'npm uninstall -g react-native react-native-cli' and try again

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 3s

    at checkExecSyncError (child_process.js:621:11)
    at execFileSync (child_process.js:639:15)
    at runOnAllDevices (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\runOnAllDevices.js:94:39)
    at buildAndRun (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\index.js:158:41)
    at C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\@react-native-community\cli-platform-android\build\commands\runAndroid\index.js:125:12
    at processTicksAndRejections (internal/process/task_queues.js:85:5)
    at async Command.handleAction (C:\Users\Master\Desktop\proj\UltimaTentativa\node_modules\react-native\node_modules\@react-native-community\cli\build\index.js:164:9)
`

## Reproducible Demo

**Comment by bneigher** [comment]
I'm seeing this too - RN 0.61.4

**Comment by bneigher** [comment]
Doing this: https://harrymoreno.com/2019/10/25/troubleshooting-react-native-androidx.html#upgrading-gradle seems to work for me

**Comment by grabbou** [comment]
@fabiocosta88, can you run

> npx.cmd --quiet --no-install react-native config' 

from `C:\Users\Master\Desktop\proj\UltimaTentativa`

and paste the output?

I wonder why Gradle says the JSON is invalid. Something must be wrong here and we need to get a reproduction to fix it for you and others!

Thanks for your collaboration

**Comment by sibelius** [comment]
it looks like, it is a permission issue

Error: EPERM: operation not permitted, scandir

check if you have permission to write in this folder

**Comment by pasine** [comment]
I'm facing a similar issue on MacOS.
Not sure if this is related, but the error is also saying `Failed to parse React Native CLI configuration`.  

I tried creating a new project from scratch.  

This is working
```
npx react-native init AwesomeProject --version 0.60.0
npx react-native run-android
```

This is not  
```
npx react-native init AwesomeProject
npx react-native run-android
```

```
Error: Command failed: ./gradlew app:installDebug -PreactNativeDevServerPort=8081
:ReactNative:Failed to parse React Native CLI configuration: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object

The current character read is 'i' with an int value of 105
Unable to determine the current character, it is not a string, number, array, or object
line number 1
index number 0
info Run "react-native --help" to see a list of all available commands.
^

FAILURE: Build failed with an exception.
```

**Comment by fabiocosta88** [comment]
Guys I just came back a backup that I had ... I don't know about the source of the error, but in this backup I just redid everything the same before and I didn't see the error again.

**Comment by sagargulati** [comment]
Any fix for this found yet?

**Comment by fgagneten** [comment]
Same issue here!

**Comment by fgagneten** [comment]
It worked for me doing this:

https://stackoverflow.com/a/59507720/6421759

**Comment by grabbou** [comment]
This sounds like something we should fix really quick. 

> :ReactNative:Failed to parse React Native CLI configuration: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object

The error is rather verbose.

Can you email me the configuration? @pasine 

Without a configuration that's leading to an error, there's not much we can do.

**Comment by croson** [comment]
@grabbou Is...solved?

**Comment by nmasnadithya** [comment]
I'm having the same issue, it's a project generated by igniter.

> This sounds like something we should fix really quick.
> 
> > :ReactNative:Failed to parse React Native CLI configuration: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object
> 
> The error is rather verbose.
> 
> Can you email me the configuration? @pasine
> 
> Without a configuration that's leading to an error, there's not much we can do.

Hope this (output from `npx --quiet --no-install react-native config`) would help. I redacted some paths to not reveal my personal info

```json
{
  "root": "<path_to_project>",
  "reactNativePath": "<path_to_project>/node_modules/react-native",
  "dependencies": {
    "@react-native-community/async-storage": {
      "root": "<path_to_project>/node_modules/@react-native-community/async-storage",
      "name": "@react-native-community/async-storage",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/@react-native-community/async-storage/ios",
          "folder": "<path_to_project>/node_modules/@react-native-community/async-storage",
          "pbxprojPath": "<path_to_project>/node_modules/@react-native-community/async-storage/ios/RNCAsyncStorage.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/@react-native-community/async-storage/RNCAsyncStorage.podspec",
          "projectPath": "<path_to_project>/node_modules/@react-native-community/async-storage/ios/RNCAsyncStorage.xcodeproj",
          "projectName": "RNCAsyncStorage.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/@react-native-community/async-storage/android",
          "folder": "<path_to_project>/node_modules/@react-native-community/async-storage",
          "packageImportPath": "import com.reactnativecommunity.asyncstorage.AsyncStoragePackage;",
          "packageInstance": "new AsyncStoragePackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "@react-native-community/masked-view": {
      "root": "<path_to_project>/node_modules/@react-native-community/masked-view",
      "name": "@react-native-community/masked-view",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/@react-native-community/masked-view/ios",
          "folder": "<path_to_project>/node_modules/@react-native-community/masked-view",
          "pbxprojPath": "<path_to_project>/node_modules/@react-native-community/masked-view/ios/RNCMaskedView.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/@react-native-community/masked-view/RNCMaskedView.podspec",
          "projectPath": "<path_to_project>/node_modules/@react-native-community/masked-view/ios/RNCMaskedView.xcodeproj",
          "projectName": "RNCMaskedView.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/@react-native-community/masked-view/android",
          "folder": "<path_to_project>/node_modules/@react-native-community/masked-view",
          "packageImportPath": "import org.reactnative.maskedview.RNCMaskedViewPackage;",
          "packageInstance": "new RNCMaskedViewPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-gesture-handler": {
      "root": "<path_to_project>/node_modules/react-native-gesture-handler",
      "name": "react-native-gesture-handler",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-gesture-handler/ios",
          "folder": "<path_to_project>/node_modules/react-native-gesture-handler",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-gesture-handler/ios/RNGestureHandler.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-gesture-handler/RNGestureHandler.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-gesture-handler/ios/RNGestureHandler.xcodeproj",
          "projectName": "RNGestureHandler.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-gesture-handler/android",
          "folder": "<path_to_project>/node_modules/react-native-gesture-handler",
          "packageImportPath": "import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;",
          "packageInstance": "new RNGestureHandlerPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-keychain": {
      "root": "<path_to_project>/node_modules/react-native-keychain",
      "name": "react-native-keychain",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-keychain",
          "folder": "<path_to_project>/node_modules/react-native-keychain",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-keychain/RNKeychain.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-keychain/RNKeychain.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-keychain/RNKeychain.xcodeproj",
          "projectName": "RNKeychain.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-keychain/android",
          "folder": "<path_to_project>/node_modules/react-native-keychain",
          "packageImportPath": "import com.oblador.keychain.KeychainPackage;",
          "packageInstance": "new KeychainPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-localize": {
      "root": "<path_to_project>/node_modules/react-native-localize",
      "name": "react-native-localize",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-localize/ios",
          "folder": "<path_to_project>/node_modules/react-native-localize",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-localize/ios/RNLocalize.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-localize/RNLocalize.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-localize/ios/RNLocalize.xcodeproj",
          "projectName": "RNLocalize.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-localize/android",
          "folder": "<path_to_project>/node_modules/react-native-localize",
          "packageImportPath": "import com.reactcommunity.rnlocalize.RNLocalizePackage;",
          "packageInstance": "new RNLocalizePackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-safe-area-context": {
      "root": "<path_to_project>/node_modules/react-native-safe-area-context",
      "name": "react-native-safe-area-context",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-safe-area-context/ios",
          "folder": "<path_to_project>/node_modules/react-native-safe-area-context",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-safe-area-context/ios/SafeAreaView.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-safe-area-context/react-native-safe-area-context.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-safe-area-context/ios/SafeAreaView.xcodeproj",
          "projectName": "SafeAreaView.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-safe-area-context/android",
          "folder": "<path_to_project>/node_modules/react-native-safe-area-context",
          "packageImportPath": "import com.th3rdwave.safeareacontext.SafeAreaContextPackage;",
          "packageInstance": "new SafeAreaContextPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-screens": {
      "root": "<path_to_project>/node_modules/react-native-screens",
      "name": "react-native-screens",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-screens/ios",
          "folder": "<path_to_project>/node_modules/react-native-screens",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-screens/ios/RNScreens.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-screens/RNScreens.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-screens/ios/RNScreens.xcodeproj",
          "projectName": "RNScreens.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-screens/android",
          "folder": "<path_to_project>/node_modules/react-native-screens",
          "packageImportPath": "import com.swmansion.rnscreens.RNScreensPackage;",
          "packageInstance": "new RNScreensPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    },
    "react-native-splash-screen": {
      "root": "<path_to_project>/node_modules/react-native-splash-screen",
      "name": "react-native-splash-screen",
      "platforms": {
        "ios": {
          "sourceDir": "<path_to_project>/node_modules/react-native-splash-screen/ios",
          "folder": "<path_to_project>/node_modules/react-native-splash-screen",
          "pbxprojPath": "<path_to_project>/node_modules/react-native-splash-screen/ios/SplashScreen.xcodeproj/project.pbxproj",
          "podfile": null,
          "podspecPath": "<path_to_project>/node_modules/react-native-splash-screen/react-native-splash-screen.podspec",
          "projectPath": "<path_to_project>/node_modules/react-native-splash-screen/ios/SplashScreen.xcodeproj",
          "projectName": "SplashScreen.xcodeproj",
          "libraryFolder": "Libraries",
          "sharedLibraries": [],
          "plist": [],
          "scriptPhases": []
        },
        "android": {
          "sourceDir": "<path_to_project>/node_modules/react-native-splash-screen/android",
          "folder": "<path_to_project>/node_modules/react-native-splash-screen",
          "packageImportPath": "import org.devio.rn.splashscreen.SplashScreenReactPackage;",
          "packageInstance": "new SplashScreenReactPackage()"
        }
      },
      "assets": [],
      "hooks": {},
      "params": []
    }
  },
  "commands": [
    {
      "name": "log-ios",
      "description": "starts iOS device syslog tail"
    },
    {
      "name": "run-ios",
      "description": "builds your app and starts it on iOS simulator",
      "examples": [
        {
          "desc": "Run on a different simulator, e.g. iPhone 5",
          "cmd": "react-native run-ios --simulator \"iPhone 5\""
        },
        {
          "desc": "Pass a non-standard location of iOS directory",
          "cmd": "react-native run-ios --project-path \"./app/ios\""
        },
        {
          "desc": "Run on a connected device, e.g. Max's iPhone",
          "cmd": "react-native run-ios --device \"Max's iPhone\""
        },
        {
          "desc": "Run on the AppleTV simulator",
          "cmd": "react-native run-ios --simulator \"Apple TV\"  --scheme \"helloworld-tvOS\""
        }
      ],
      "options": [
        {
          "name": "--simulator [string]",
          "description": "Explicitly set simulator to use. Optionally include iOS version betweenparenthesis at the end to match an exact version: \"iPhone 6 (10.0)\"",
          "default": "iPhone 11"
        },
        {
          "name": "--configuration [string]",
          "description": "Explicitly set the scheme configuration to use",
          "default": "Debug"
        },
        {
          "name": "--scheme [string]",
          "description": "Explicitly set Xcode scheme to use"
        },
        {
          "name": "--project-path [string]",
          "description": "Path relative to project root where the Xcode project (.xcodeproj) lives.",
          "default": "ios"
        },
        {
          "name": "--device [string]",
          "description": "Explicitly set device to use by name.  The value is not required if you have a single device connected."
        },
        {
          "name": "--udid [string]",
          "description": "Explicitly set device to use by udid"
        },
        {
          "name": "--no-packager",
          "description": "Do not launch packager while building"
        },
        {
          "name": "--verbose",
          "description": "Do not use xcpretty even if installed"
        },
        {
          "name": "--port [number]",
          "default": 8081
        },
        {
          "name": "--terminal [string]",
          "description": "Launches the Metro Bundler in a new window using the specified terminal path."
        }
      ]
    },
    {
      "name": "log-android",
      "description": "starts logkitty"
    },
    {
      "name": "run-android",
      "description": "builds your app and starts it on a connected Android emulator or device",
      "options": [
        {
          "name": "--root [string]",
          "description": "Override the root directory for the android build (which contains the android directory)",
          "default": ""
        },
        {
          "name": "--variant [string]",
          "description": "Specify your app's build variant",
          "default": "debug"
        },
        {
          "name": "--appFolder [string]",
          "description": "Specify a different application folder name for the android source. If not, we assume is \"app\"",
          "default": "app"
        },
        {
          "name": "--appId [string]",
          "description": "Specify an applicationId to launch after build.",
          "default": ""
        },
        {
          "name": "--appIdSuffix [string]",
          "description": "Specify an applicationIdSuffix to launch after build.",
          "default": ""
        },
        {
          "name": "--main-activity [string]",
          "description": "Name of the activity to start",
          "default": "MainActivity"
        },
        {
          "name": "--deviceId [string]",
          "description": "builds your app and starts it on a specific device/simulator with the given device id (listed by running \"adb devices\" on the command line)."
        },
        {
          "name": "--no-packager",
          "description": "Do not launch packager while building"
        },
        {
          "name": "--port [number]",
          "default": 8081
        },
        {
          "name": "--terminal [string]",
          "description": "Launches the Metro Bundler in a new window using the specified terminal path."
        },
        {
          "name": "--tasks [list]",
          "description": "Run custom Gradle tasks. By default it's \"installDebug\""
        },
        {
          "name": "--no-jetifier",
          "description": "Do not run \"jetifier\" – the AndroidX transition tool. By default it runs before Gradle to ease working with libraries that don't support AndroidX yet. See more at: https://www.npmjs.com/package/jetifier.",
          "default": false
        }
      ]
    }
  ],
  "assets": [
    "<path_to_project>/app/theme/fonts/Montserrat-Black.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-BlackItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Bold.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-BoldItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-ExtraBold.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-ExtraBoldItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-ExtraLight.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-ExtraLightItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Italic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Light.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-LightItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Medium.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-MediumItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Regular.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-SemiBold.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-SemiBoldItalic.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-Thin.ttf",
    "<path_to_project>/app/theme/fonts/Montserrat-ThinItalic.ttf"
  ],
  "platforms": {
    "ios": {},
    "android": {}
  },
  "haste": {
    "providesModuleNodeModules": [
      "react-native"
    ],
    "platforms": [
      "ios",
      "android"
    ]
  },
  "project": {
    "ios": {
      "sourceDir": "<path_to_project>/ios",
      "folder": "<path_to_project>",
      "pbxprojPath": "<path_to_project>/ios/rNativeTest.xcodeproj/project.pbxproj",
      "podfile": "<path_to_project>/ios/Podfile",
      "podspecPath": null,
      "projectPath": "<path_to_project>/ios/rNativeTest.xcodeproj",
      "projectName": "rNativeTest.xcodeproj",
      "libraryFolder": "Libraries",
      "sharedLibraries": [],
      "plist": [],
      "scriptPhases": []
    },
    "android": {
      "sourceDir": "<path_to_project>/android/app",
      "isFlat": false,
      "folder": "<path_to_project>",
      "stringsPath": "<path_to_project>/android/app/src/main/res/values/strings.xml",
      "manifestPath": "<path_to_project>/android/app/src/main/AndroidManifest.xml",
      "buildGradlePath": "<path_to_project>/android/app/build.gradle",
      "settingsGradlePath": "<path_to_project>/android/settings.gradle",
      "assetsPath": "<path_to_project>/android/app/src/main/assets",
      "mainFilePath": "<path_to_project>/android/app/src/main/java/com/rnativetest/MainApplication.java",
      "packageName": "com.rnativetest"
    }
  }
}
```

**Comment by tusharsinghal1610** [comment]
delete the files it was finding error as  "The current character read is 'E' with an int value of 69"
Deleting app\build\intermediates\signing_config\debug\out\signing-config.json
worked for me.

**Comment by sewardj1011** [comment]
I had this same issue and nothing seemed  to work for me until I found the signing-config file in "<Project-Root>\android\app\build\intermediates\signing_config\debug\out" and changed the permissions to allow all users in Windows to modify the file. Once done everything worked like a charm for me. Hope this helps!

**Comment by grabbou** [comment]
It is really strange. I am going to pin this to "known issues" until we figure out a better strategy on how to move further.

Not really sure why signing configuration (and its permissions) would be an issue here. We're shipping better error messages for this one, hopefully it helps to find the root cause.

**Comment by grabbou** [comment]
I can't really reproduce this locally, so we're going to ship a better error message that actually prints the output of the command that we are unable to parse.

That way, we will see where does it actually fail.

**Comment by grabbou** [comment]
I have released a `3.0.5-alpha.0` of the CLI. 

Please use [Yarn resolutions](https://legacy.yarnpkg.com/en/docs/selective-version-resolutions/) mechanism to overwrite the version you have and let me know what is the error message you're getting. I appreciate your help!

**Comment by lucasluca** [comment]
Just removed the build folder inside android and go to android folder and then in powershell typed ./gradlew clean and then  cd.. => npx react-native run-android

**Comment by traxx10** [comment]
I have the same issue.
I created a new App after the previous one had the same issue now this guy is throwing the same issue too.

**Comment by ShaMan123** [comment]
#902 related?

**Comment by thymikee** [comment]
This is one of the few errors connected to the same problem, and we already have duplicate issues around it. We've been tackling this from few sides recently and we believe it's fixed now. 
Please [update the `@react-native-community/cli`](https://github.com/react-native-community/cli#updating-the-cli) to the latest version.

**Comment by ashadnasim52** [comment]
I solved by checking AndroidManifest.xml and solving the errors there.
Basically there has the error in my case.
and also deleted node_modules and run 
```
yarn install
```

**Comment by SolutionPr** [comment]
I'm having the same issue, during generated singed apk in react native
please help me and provide me solution of this issues 

FAILURE: Build failed with an exception.

* Where:
Script 'C:\Nodejs_Tricky\Freshser-Project\freshr-app\node_modules\@react-native-community\cli-platform-android\native_modules.gradle' line: 426

* What went wrong:
A problem occurred evaluating script.
> Calling `[node, C:\Nodejs_Tricky\Freshser-Project\freshr-app\node_modules\@react-native-community\cli\build\bin.js, config]` finished with an exception. Error message: groovy.json.JsonException: Unable to determine the current character, it is not a string, number, array, or object

  The current character read is 'E' with an int value of 69
  Unable to determine the current character, it is not a string, number, array, or object
  line number 1
  index number 0


![image](https://github.com/react-native-community/cli/assets/93113701/0be38fbe-4ac3-4924-9e07-6afb2102f5b2)

**Comment by adamTrz** [comment]
@SolutionPr  - in logs I can see an error saying `Package name not found in ${manifestPath}`. 
I would assume that your package name lives in `build.gradle` file? That was fixed with #1800, where we now search there as well. 

Maybe try to upgrade react-native version and CLI or try to fix AndroidManifest and add package name there as well?
